bodhi/server/models.py

```diff
@@ -923,6 +933,7 @@ class Update(Base):
     status = Column(UpdateStatus.db_type(),
                     default=UpdateStatus.pending,
                     nullable=False)
+    ci_status = Column(UpdateCiStatus.db_type(), default=None, nullable=True)
```

Comment:
Given that the column is nullable, does it need a default of ``None``? Shouldn't that be a no-op?